sdk/communication/azure-communication-administration/samples/phone_number_area_codes_sample.py

```diff
@@ -0,0 +1,45 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: phone_number_area_codes_sample.py
+DESCRIPTION:
+    This sample demonstrates how to get all area codes via a connection string, country code and phone plan id.
+USAGE:
+    phone_number_area_codes_sample.py
```

Comment:
I think you want to have "python phone_number_area_codes_sample.py"?

Reply:
yes, you're right. will fix it, thanks!